docs/installation_configuration_guide/Installation.md

```diff
@@ -9,7 +9,74 @@ documentation for details on how to configure them for optimal performance .
 ## Docker based installation
                     
 Nussknacker is available at [Docker hub](https://hub.docker.com/r/touk/nussknacker/). 
-To see how to install needed components 
+You can check an example usage with docker-compose at [Nussknacker Quickstart Repository](https://github.com/TouK/nussknacker-quickstart).
+
+### Base Image
+
+As a base image we uses `openjdk:11-jdk`. See [Open JDK's Docker hub](https://hub.docker.com/_/openjdk) for more details.
+
+### Properties
+
+Available Nussknacker image properties:
+
+| Property name  | Type     | Default value | Description
```

Comment:
Are these properties of docker image, or of default configuration? Some of these are docker-specific, but most are from defaultUiConfig - maybe we can keep those separately/distinguished as they are the same for e.g. tgz distribution?